scripts/src/owners/checkuser.py

```diff
@@ -0,0 +1,68 @@
+import re
+import argparse
+import requests
+import os
+import sys
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+def verify_user(username):
+    print(f"[INFO] Verify user. {username}")
+    owners_path = "OWNERS"
```

Comment:
minor: define constant instead of string literals as we used `re.compile(r"OWNERS")` in the following code.

Reply:
Done.